docs/source/release_notes.rst

```diff
@@ -8,11 +8,18 @@ Future Release
         * Add ``deep`` parameter to Woodwork Accessor and Schema equality checks (:pr:`889`)
     * Fixes
     * Changes
+        * Remove command line functions for list logical and semantic tags (:pr:`891`)
```

Comment:
did you mean "for list logical types and semantic tags"?

Reply:
Fixed.